lib/rules/selector-max-pseudo-class/index.js

```diff
@@ -71,14 +76,14 @@ function rule(max) {
 
 			for (const selector of ruleNode.selectors) {
 				for (const resolvedSelector of resolvedNestedSelector(selector, ruleNode)) {
-					parseSelector(resolvedSelector, result, rule, (selectorTree) => {
+					parseSelector(resolvedSelector, result, ruleNode, (selectorTree) => {
```

Comment:
[note] This is a potential bug, but it's an edge case. Because `rule` (`node) is used only when a parse error occurs.

https://github.com/stylelint/stylelint/blob/6df0c6bbd6cb59e0f2027b3a21609bb39cf1bacf/lib/utils/parseSelector.js#L15-L17